amptorch/tests/consistency_test.py

```diff
@@ -121,7 +120,7 @@ def test_energy_force_consistency():
     torch_Cu_weights_2 = torch.FloatTensor(Cu_weights_2[:-1, :]).t()
     torch_Cu_bias_2 = torch.FloatTensor(Cu_weights_2[-1, :])
 
-    descriptor_setup = ("gaussian", Gs, elements)
+    descriptor_setup = ("gaussian", Gs, {"cutoff_func": "cosine"}, elements)
```

Comment:
This descriptor setup is starting to look a little awkward with the nested dictionary. Can we do this differently/cleaner so it's more clear what things are?

```
descriptor_setup  = {
    "descriptor": Gaussian,
    "params": Gs,
    "cutoff_func": ...
    "elements":...
}
```